codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/util/Smithy.kt

```diff
@@ -152,3 +152,21 @@ fun ServiceShape.serviceNameOrDefault(default: String) = getTrait<TitleTrait>()?
 
 /** Returns the SDK ID of the given service shape */
 fun ServiceShape.sdkId(): String = getTrait<ServiceTrait>()?.sdkId?.lowercase()?.replace(" ", "") ?: id.getName(this)
+
+/**
+ * A set of RPC-bound Smithy protocols
+ */
+val RPC_BOUND_PROTOCOLS =
```

Comment:
This is not open to extension for someone else to write a new protocol that doesn't live in smithy-rs.

Reply:
Certainly not, should've caught that in the first place. Fixed in 75989c3.

Reply:
Sorry what I meant was not to make it private so much as providing a way for someone implementing a protocol to express this without us having to update the list.